lang/en.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
I updated this to `ts` file and added a nocheck so we can start using it.  I converted it to a `js` originally which was short sighted because we can't use import/exports without flipping the project type to module.

Reply:
Because it has javascript functions inside of it.  So it wants those all to be typed 😞